src/plugins/data/public/shard_failure_modal/shard_failure_description.tsx

```diff
@@ -96,7 +96,7 @@ export function ShardFailureDescription(props: ShardFailure) {
       <EuiFlexItem grow={false}>
         <EuiDescriptionList
           type="responsiveColumn"
-          gutterSize="s"
+          rowGutterSize="s"
```

Comment:
We can technically remove these props completely now that `rowGutterSize` defaults to `"s"`.

@kyrspl what was the intention of the breaking `EuiDescriptionList` gutter size change? Do you want all downstream instances in Kibana to adopt the new default gutter size, or should we update old usages to set `<EuiDescriptionList size="m">` to preserve the old gutter size?

Reply:
Visual preference - if we think it's going to break a lot of instances let's switch it to `m` and we can review later.

Reply:
@kyrspl and I discussed this as the beginning of my workday Tuesday (Aug 29). We decided it would be better to update the `EuiDescriptionList` props `rowGutterSize` and `columnGutterSize` to have default `m`. I'll update the component in EUI, issue a new pull request, then make a minor point release and fold it into this Kibana upgrade.

That way we're keeping the default the same as was prior for row margins, and honoring the local Kibana overrides by passing in `s`.

/cc @cee-chen

Reply:
Upstream bump to EUI: https://github.com/elastic/eui/pull/7140